ac_runtime-api/modules/txpayment.adoc

```diff
@@ -73,3 +75,67 @@ where::::
 *** latexmath:[f_l] is the length fee, the amount paid for the encoded length (in bytes) of the extrinsic.
 *** latexmath:[f_a] is the "`adjusted weight fee`", which is a multiplication of the fee multiplier and the weight fee. The fee multiplier varies depending on the usage of the network.
 ** latexmath:[t] is the tip for the block author.
+
+[#sect-rte-transactionpaymentapi-query-call-info]
+==== `TransactionPaymentApi_query_call_info`
+
+Query information of a dispatch class, weight, and fee of a given encoded `Call`.
+
+Arguments::
+* A byte array of varying sizes containing the `Call`.
```

Comment:
We always avoided using overly Rust/Substrate specific concepts like `Call`. Simply refer to it as "extrinsic" (and for all other mentions of `Call`).

Reply:
There seems to be a difference between a runtime call and an extrinsic. Could you please provide any link where this is evident? the `transactionpaymentapi_query_info` and `transactionpaymentapi_query_fee_details` already exist for extrinsics. The new APIs `transactionpaymentapi_query_call_info` and `transactionpaymentapi_query_call_fee_details` are separately declared, however the type of RuntimeCall and Extrinsic seem to be the same from substrate code. We may need to add a definition for `Runtime Call` and explain how they are different from Extrinsics.